src/moudels/Worker.tsx

```diff
@@ -5,7 +5,7 @@ export interface Worker {
   title: string;
   country: string;
   city: string;
-  birthDate: Date;
+  birthDate: string;
```

Comment:
Why save it as a string and not as a date like it was before? Date is safer to use than string